Readme.md

```diff
@@ -1,23 +1,24 @@
-# The FlexMeasures Platform
+![FlexMeasures Logo](https://artwork.lfenergy.org/projects/flexmeasures/horizontal/color/flexmeasures-horizontal-color.png)
```

Comment:
https://stackoverflow.com/questions/65413712/changing-readme-md-image-display-conditional-to-github-light-mode-dark-mode

Reply:
Honestly, I think this logo looks okay in the Readme on dark mode.

Reply:
Moved to https://github.com/FlexMeasures/flexmeasures/issues/420